src/components/DisplayCurrentPlaylist.js

```diff
@@ -0,0 +1,40 @@
+/*
+
+  This Component displays the current playlists.
+
+  props - current playlist , allSongs
+*/
+
+import { playlistType, songType } from "../lib/types.js";
+import PropTypes from "prop-types";
+import styles from "../styles/DisplayCurrentPlaylist.module.css";
+
+export default function DisplayCurrentPlaylist({ playlist, allSongs }){
+
+const currentSongs = allSongs.filter((song) => song.playlistID === playlist.id);
+
+currentSongs.map((song) => song.timeAdded = new Date(song.timeAdded));
+
+const currentPlaylist = currentSongs.map((song) => 
```

Comment:
This looks like you're creating an array of unordered lists - I think these should be `<li>`s.

Also, this isn't a pressing issue but we may want to standardize how we deal with dates and just use the moment library for everything. Here is a cool [cheatsheet](https://devhints.io/moment) that I found. Perhaps we should store the song times in a different format to make this easier - we can talk about this as a group.

Reply:
I agree, made the change to `<li>`s. I would love to talk about dates and moment, I saw you're update in either `PlaylistDetails` or `ShowDetails` and had the same thought. That being said, the cheatsheet was what I needed. I changed the `SongInput` to the format type (`.format("LT")`)we were trying to achieve in the `DisplayCurrentPlaylist` component. We were trying to achieve standard 12 hour time (ex: 8:42 am), we can talk about if we want to display in a different format